.gitignore

```diff
@@ -1,8 +1,20 @@
 # Dataset-specific files
-api/dataset_config/*
+api/dataset_config/**
 !api/dataset_config/test_dataset
 api/app.yaml
 
+!api/dataset_config/template
+
+# Most Platinum Genomes config files are in data-explorer-indexers repo. 
+# (ui.json is in this repo since it's only used by this repo.)
+# Only store ui.json; eg we don't want to maintain 2 copies of dataset.json.
+# Negating a previously excluded directory is tricky. See
+# https://stackoverflow.com/a/18702826
+# Note that above must have api/dataset_config/** and not api/dataset_config/*
+!api/dataset_config/platinum_genomes/
```

Comment:
Shouldn't this directory be ignored excluding the two files below? So it should be:

```
api/dataset_config/**
!api/dataset_config/platinum_genomes/ui.json
!api/dataset_config/platinum_genomes/README.md
```

Reply:
Ah okay, the `!api/dataset_config/platinum_genomes/` is kind of unintuitive but as long as I can't check in `platinum_genomes/random_file` I'm ok with this